selfdrive/swaglog.py

```diff
@@ -22,11 +22,10 @@ def emit(self, record):
       self.connect()
 
     msg = self.format(record).rstrip('\n')
-    # print "SEND", repr(msg)
     try:
       self.sock.send(chr(record.levelno)+msg, zmq.NOBLOCK)
     except zmq.error.Again:
-      # drop :/
+      # If we are unable to send the message, we are as of now forced to drop it
```

Comment:
It doesn't really, but looking at the code for the first time It wasn't especially clear to me what exactly was being dropped so I thought it might be useful to clarify that to someone on a first pass.

That could also just be idiocy on my part.